docs/introduction/required-php-configuration.md

```diff
@@ -65,7 +65,7 @@ opcache.use_cwd=1
 | pdo            | required by package `doctrine/orm`                                                                                                                            |
 | pdo_pgsql      | required to support `pdo_pgsql` database driver                                                                                                               |
 | pgsql          | used in acceptance for fast repopulating of database using `COPY` command                                                                                     |
-| redis          | required by package `snc/redis-bundle` and sessions stored in Redis                                                                                           |
+| redis          | required by package `snc/redis-bundle` and sessions stored in Redis (required version is 4.1.1 because of lazyloading)                                        |
```

Comment:
```suggestion
| redis          | required by package `snc/redis-bundle` and sessions stored in Redis (minimal version is 4.1.1 because of lazyloading)                                        |
```